extension/intents/clipboard/clipboard.js

```diff
@@ -71,6 +71,14 @@ intentRunner.registerIntent({
   },
 });
 
+intentRunner.registerIntent({
+  name: "clipboard.copyImageLink",
```

Comment:
One other thing – this (I believe) is copying the image to the clipboard, not a link to the image. So the intent should be named accordingly, `"clipboard.copyImage"`